### ISSUE TITLE
alternative Mapping-Projekte

### ISSUE THREAD

**Issue by andneuma** [kjoscha/lichtenberg#37]
Jo, also wie gesagt, hier gibt es ein Alternativprojekt: http://refugeeswelcomemap.de/

Unterschiede +/- zu unserem Projekt nach oberflächlichem Ansehen:
+ Es steht bereits alles inkl. bundesweitem Fokus
+ Ebenso (einigermaßen) partizipativ

- Mehrsprachigkeit
- Mobil-Version ist nicht funktionabel (zum. auf meinem Handy)
- Kategorien beziehen sich lediglich auf die Standard-Sachen ("Hilfe", "Ämter", "Sonstiges")

Würde sagen, dass das ein ernstzunehmendes Alternativprojekt ist und wir damit umgehen müssen, dass Leute uns da technisch bereits ein gutes Stück voraus sind, aber auch hinter dem Liegen, was wir bzw. du gebaut hast. Es gibt von hier aus zwei Möglichkeiten

1. Wir ziehen unseren Stiefel durch
2. Wir kooperieren mit denen und mapfix und bauen deren Plattform auf mobil und mehrsprachig sowie mehr Kategorien um

Was meinst du? Im Endeffekt ist es tragisch, dass man nicht bereits an früherer Stelle von derartigen Projekten mitbekommt und so parallel vor sich hindübelt anstatt gemeinsam :/

**Comment by kjoscha** [comment]
Jow, das ist ja quasi das Gleiche in grün (nee, sogar die Farbpalette ist ähnlich...). Hmm, Mehrsprachigkeit und Mobilversion sind natürlich schon wichtig. Da wir ja eh mit MapFix kooperieren wollen, kann ich ja direkt mit denen besprechen, wie wir mit dem Projekt umgehen wollen, oder?

**Comment by kjoscha** [comment]
Die Seite hat halt genau die Elemente, die wir noch nicht haben (z.B. mehrere Städte anfahrbar, schöne POI-Listen), dafür aber keine Mehrsprachigkeit und Mobilversion. Im Prinzip wäre ein Mergen beider Projekte perfekt. Aber ist dann auch eine Frage der technischen Praktikabilität...

**Comment by andneuma** [comment]
Jo, Merten wäre super, die haben halt auch bereits einen guten datenstock. Außerdem sollte man eigentlich die Leute von arriving in Berlin da mit einbinden. Sie schreiben auf ihrere webseite, dass sie an mehrsprachigkeit arbeiten, deine Umsetzung lässt sich da sicherlich mit einbringen. Insbesondere das crowdsourcing-ding ist in hinblick auf Übersetzung cool, das sollte man stark machen. Ich könnte übernehmen die mal an zuschreiben, dann kannst du für das treffen am fr gleich was mitnehmen.

**Comment by kjoscha** [comment]
jow super

**Comment by magdalena19** [comment]
>Im Endeffekt ist es tragisch, dass man nicht bereits an früherer Stelle von derartigen Projekten mitbekommt und so parallel vor sich hindübelt anstatt gemeinsam :/

Wollte dazu nur nochmal betonen, dass es zumindest meinerseits kein Problem ist, da ich jetzt schon extrem viel gelernt habe, Spaß dabei hatte und man die Software - falls wir sie nicht weiterführen - ausbauen könnte zu einer Art Startpaket für andere partizipatorische, multilinguale Mappingprojekte...

Darüber hinaus ist es in der Software-Welt ja ansonsten auch üblich, dass zu Hauf parallel entwickelt wird und später entweder kooperiert wird oder sich eine Software durchsetzt. Das macht es natürlich nicht weniger absurd und wir befinden uns hier ja auch nicht im Wettbewerb, aber wir sollten uns auf jeden Fall nicht ärgern. So wie es aussieht, schaffen wir es gerade ja ganz gut, das Gesamtprojekt (im Sinne aller Projekte und der ) nach vorne zu bringen, das ist doch cool!
